code/modules/mob/living/simple_animal/friendly/corgi.dm

```diff
@@ -162,7 +162,8 @@
 					return
 				new /mob/living/simple_animal/corgi/puppy(loc)
 
-
-		if(prob(1))
-			visible_emote(pick("dances around","chases her tail"),0)
-			INVOKE_ASYNC(src, .proc/do_dance, list(1,2,4,8,4,2,1,2,4,8,4,2,1,2,4,8,4,2))
+/mob/living/simple_animal/corgi/Lisa/think()

```

Comment:
Should Lisa multiplying be under AI? I think so, as it's something the player doesn't have control over. Refer to lines 144 - 163.